### ISSUE TITLE
[Bug]:  Error “Your storage is full, files can not be updated or synced anymore!"

### ISSUE THREAD

**Issue by lanzi69** [nextcloud/server#37940]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [ ] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Hey there,

my Nextcloud Hub 4 (26.0.1) is running on a Synology NAS containing 4 TB of disc space (about 2 TB free).

Last weekend I updated from 24.0.2 to 26.0.1 and after that the web-frontend said, I'm using 97% of my available storage.

My user is configured with unlimited storage quota and meanwhile the frontend shows the above "storage is full" message.

It displays a usage of 64.3 GB but in a Linux shell on the NAS the "du -sh" command says, that in fact there are 205 GB used.

Rgds
Lanzi

### Steps to reproduce

1. Log in to web-frontend.

### Expected behavior

No error or warning to storage issues, because there's unlimited quota an lot's of free space on disc.

### Installation method

Community Manual installation with Archive

### Nextcloud Server version

26

### Operating system

Other

### PHP engine version

PHP 8.0

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 22.2.3 to 23.0.1)

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "***REMOVED SENSITIVE VALUE***",
            "192.168.0.4"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "26.0.1.1",
        "overwrite.cli.url": "https:\/\/***REMOVED SENSITIVE VALUE***",
        "overwriteprotocol": "https",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": false,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "defaultapp": "files",
        "default_phone_region": "DE",
        "logtimezone": "Europe\/Berlin",
        "installed": true,
        "default_language": "de",
        "force_language": "de",
        "default_locale": "de_DE",
        "force_locale": "de_DE",
        "maintenance": false,
        "updatechecker": false,
        "theme": "",
        "loglevel": 0,
        "logfile": "\/volume1\/Nextcloud\/nextcloud.log",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379
        },
        "updater.release.channel": "stable",
        "mail_smtpmode": "smtp",
        "mail_smtpsecure": "tls",
        "app_install_overwrite": [
            "occweb"
        ],
        "data-fingerprint": "85cdc76e9c26e524d87ed711464c2411",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_sendmailmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "587",
        "mail_smtpauth": 1,
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauthtype": "LOGIN",
        "activity_expire_days": 32
    }
}
```

### List of activated Apps

```shell
Enabled:
  - admin_audit: 1.16.0
  - calendar: 4.3.3
  - cloud_federation_api: 1.9.0
  - comments: 1.16.0
  - contacts: 5.2.0
  - contactsinteraction: 1.7.0
  - dav: 1.25.0
  - federatedfilesharing: 1.16.0
  - federation: 1.16.0
  - files: 1.21.1
  - files_external: 1.18.0
  - files_rightclick: 1.5.0
  - files_sharing: 1.18.0
  - files_trashbin: 1.16.0
  - files_versions: 1.19.1
  - logreader: 2.11.0
  - lookup_server_connector: 1.14.0
  - notes: 4.7.2
  - oauth2: 1.14.0
  - password_policy: 1.16.0
  - previewgenerator: 5.2.4
  - privacy: 1.10.0
  - provisioning_api: 1.16.0
  - qownnotesapi: 23.3.0
  - related_resources: 1.1.0-alpha1
  - serverinfo: 1.16.0
  - settings: 1.8.0
  - support: 1.9.0
  - systemtags: 1.16.0
  - text: 3.7.2
  - theming: 2.1.1
  - twofactor_backupcodes: 1.15.0
  - updatenotification: 1.16.0
  - viewer: 1.10.0
  - workflowengine: 2.8.0
Disabled:
  - activity: 2.18.0 (installed 2.9.1)
  - bruteforcesettings: 2.6.0 (installed 1.0.3)
  - circles: 26.0.0 (installed 23.0.0)
  - dashboard: 7.6.0 (installed 7.0.0)
  - encryption: 2.14.0
  - files_pdfviewer: 2.7.0 (installed 1.2.1)
  - firstrunwizard: 2.15.0 (installed 2.5.0)
  - nextcloud_announcements: 1.15.0 (installed 1.7.0)
  - notifications: 2.14.0 (installed 2.13.1)
  - photos: 2.2.0 (installed 1.5.0)
  - recommendations: 1.5.0 (installed 0.4.0)
  - sharebymail: 1.16.0 (installed 1.2.0)
  - survey_client: 1.14.0 (installed 1.8.0)
  - suspicious_login: 4.4.0
  - twofactor_totp: 8.0.0-alpha.0
  - user_ldap: 1.16.0
  - user_status: 1.6.0 (installed 1.0.0)
  - weather_status: 1.6.0 (installed 1.0.0)
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

```shell
{"reqId":"V0QcEQuSsHU4s2UarsIF","level":0,"time":"2023-04-27T00:39:52+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/index.php/settings/admin/logging","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"TEw9f7dLRtfRoExooXaw","level":0,"time":"2023-04-27T00:39:52+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/index.php/apps/files/api/v1/stats","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"V0QcEQuSsHU4s2UarsIF","level":0,"time":"2023-04-27T00:39:54+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"logreader","method":"GET","url":"/index.php/settings/admin/logging","message":"Could not find resource file \"/apps/logreader/css/../js/logreader-main.css\"","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"logreader"}}
{"reqId":"nzy6kDFrQ5aAPhqKKQ3b","level":0,"time":"2023-04-27T00:39:54+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/index.php/js/core/merged-template-prepend.js?v=e6df84d9-0","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"aBMhYpScubtGpRpc9t84","level":0,"time":"2023-04-27T00:39:55+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/ocs/v2.php/search/providers?from=%2Fsettings%2Fadmin%2Flogging","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"XDQZfWU0CdTRqWr29KG9","level":0,"time":"2023-04-27T00:39:55+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/index.php/apps/logreader/settings","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"8cayKPbs0mCBf6xVuRFN","level":0,"time":"2023-04-27T00:39:56+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/cron.php","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"8cayKPbs0mCBf6xVuRFN","level":0,"time":"2023-04-27T00:39:56+02:00","remoteAddr":"87.***.***.***","user":"--","app":"cron","method":"GET","url":"/cron.php","message":"WebCron call has selected job with ID 8565","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"cron"}}
{"reqId":"8cayKPbs0mCBf6xVuRFN","level":0,"time":"2023-04-27T00:39:56+02:00","remoteAddr":"87.***.***.***","user":"--","app":"cron","method":"GET","url":"/cron.php","message":"Run OC\\Command\\CommandJob job with ID 8565","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"cron"}}
{"reqId":"8cayKPbs0mCBf6xVuRFN","level":0,"time":"2023-04-27T00:39:56+02:00","remoteAddr":"87.***.***.***","user":"--","app":"cron","method":"GET","url":"/cron.php","message":"Finished OC\\Command\\CommandJob job with ID 8565 in 0 seconds","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"cron"}}
{"reqId":"sbdi1HDsAyrfliRXjf6b","level":0,"time":"2023-04-27T00:39:56+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/index.php/apps/logreader/get?offset=0&count=50&levels=00111","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"05jYI5wt4cASOF2xr8nH","level":0,"time":"2023-04-27T00:40:17+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/index.php","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"mu68VGosRKDDMABm7gD6","level":0,"time":"2023-04-27T00:40:17+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/index.php/apps/files/","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"XzpuRZKj2UwD2fKppQJ1","level":0,"time":"2023-04-27T00:40:17+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/index.php","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"mu68VGosRKDDMABm7gD6","level":1,"time":"2023-04-27T00:40:17+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"no app in context","method":"GET","url":"/index.php/apps/files/","message":"Deprecated event type for \\OCP\\Collaboration\\Resources::loadAdditionalScripts: null","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":[]}
{"reqId":"mu68VGosRKDDMABm7gD6","level":1,"time":"2023-04-27T00:40:17+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"no app in context","method":"GET","url":"/index.php/apps/files/","message":"Deprecated event type for OCA\\Files::loadAdditionalScripts: Symfony\\Component\\EventDispatcher\\GenericEvent is used","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":[]}
{"reqId":"IooqIftxisNUfePEnJeB","level":0,"time":"2023-04-27T00:40:18+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/index.php/apps/files/","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"IooqIftxisNUfePEnJeB","level":1,"time":"2023-04-27T00:40:18+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"no app in context","method":"GET","url":"/index.php/apps/files/","message":"Deprecated event type for \\OCP\\Collaboration\\Resources::loadAdditionalScripts: null","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":[]}
{"reqId":"IooqIftxisNUfePEnJeB","level":1,"time":"2023-04-27T00:40:18+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"no app in context","method":"GET","url":"/index.php/apps/files/","message":"Deprecated event type for OCA\\Files::loadAdditionalScripts: Symfony\\Component\\EventDispatcher\\GenericEvent is used","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":[]}
{"reqId":"vEPolGsLemrV4GzUWdil","level":0,"time":"2023-04-27T00:40:21+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/ocs/v2.php/search/providers?from=%2Fapps%2Ffiles%2F","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"cUT1COjY5QUXLNruI0pU","level":0,"time":"2023-04-27T00:40:21+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/cron.php","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"cUT1COjY5QUXLNruI0pU","level":0,"time":"2023-04-27T00:40:21+02:00","remoteAddr":"87.***.***.***","user":"--","app":"cron","method":"GET","url":"/cron.php","message":"WebCron call has selected job with ID 8566","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"cron"}}
{"reqId":"cUT1COjY5QUXLNruI0pU","level":0,"time":"2023-04-27T00:40:21+02:00","remoteAddr":"87.***.***.***","user":"--","app":"cron","method":"GET","url":"/cron.php","message":"Run OC\\Command\\CommandJob job with ID 8566","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"cron"}}
{"reqId":"cUT1COjY5QUXLNruI0pU","level":0,"time":"2023-04-27T00:40:21+02:00","remoteAddr":"87.***.***.***","user":"--","app":"cron","method":"GET","url":"/cron.php","message":"Finished OC\\Command\\CommandJob job with ID 8566 in 0 seconds","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"cron"}}
{"reqId":"An4iF5w3VmJC6yzenDhq","level":0,"time":"2023-04-27T00:40:21+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/index.php/apps/files_sharing/api/externalShares","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"16nNkslxZ4STZLBwF6XZ","level":0,"time":"2023-04-27T00:40:22+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/ocs/v2.php/apps/text/workspace?path=%2F","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"4843ntNVQAmzBnN9FWsn","level":0,"time":"2023-04-27T00:40:22+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/index.php/apps/systemtags/lastused","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"CdST4McgwNI8XQ6eGV48","level":0,"time":"2023-04-27T00:40:23+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/index.php/apps/files/ajax/getstoragestats?dir=%2F","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"CdST4McgwNI8XQ6eGV48","level":1,"time":"2023-04-27T00:40:24+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"no app in context","method":"GET","url":"/index.php/apps/files/ajax/getstoragestats?dir=%2F","message":"Deprecated event type for \\OCP\\Collaboration\\Resources::loadAdditionalScripts: null","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":[]}
{"reqId":"CdST4McgwNI8XQ6eGV48","level":1,"time":"2023-04-27T00:40:24+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"no app in context","method":"GET","url":"/index.php/apps/files/ajax/getstoragestats?dir=%2F","message":"Deprecated event type for OCA\\Files::loadAdditionalScripts: Symfony\\Component\\EventDispatcher\\GenericEvent is used","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":[]}
{"reqId":"hMFL3K9scfPIl3OzYy5Q","level":0,"time":"2023-04-27T00:40:24+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/ocs/v1.php/apps/files_external/api/v1/mounts?format=json","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"UAaAnC5FklIDE8eovO3O","level":0,"time":"2023-04-27T00:40:24+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/index.php/apps/files_external/userglobalstorages/36?testOnly=false","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"Nz1dkUXKer2KAz25scl3","level":0,"time":"2023-04-27T00:40:24+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/index.php/apps/files_external/userglobalstorages/45?testOnly=false","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
{"reqId":"OxvIf17vI6Fd5EjmfSY1","level":0,"time":"2023-04-27T00:40:24+02:00","remoteAddr":"87.***.***.***","user":"lanzi","app":"qownnotesapi","method":"GET","url":"/index.php/apps/files_external/userglobalstorages/48?testOnly=false","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","data":{"app":"qownnotesapi"}}
```

### Additional info

https://help.nextcloud.com/t/ihr-speicher-ist-voll-daher-konnen-keine-dateien-mehr-aktualisiert-oder-synchronisiert-werden/125051/26

**Comment by joshtrichards** [comment]
What's your partitioning structure like on the NAS? In particular, which partition is your NC data Docker volume located on?

The output of a `du -shx` from within your NC data directory and a `df -h` would also be informative.

Also I'd be curious to know what the free space reported under the Disk heading under Admin->System in the NC Web UI is.

**Comment by lanzi69** [comment]
There's one large logical volume `/dev/vg1000/lv` over the hole available disc space and this volume is named `/volume1`.

These are the results of `du -shx` 

`root@nextcloud(pts/1):/volume1/Nextcloud# du -shx       `
`572G    .                                               `

... and `df -h`

`root@nextcloud(pts/1):/volume1/Nextcloud# df -h /volume1`
`Filesystem      Size  Used Avail Use% Mounted on        `
`/dev/vg1000/lv  3.6T  2.4T  1.3T  65% /volume1          `

Here's a screenshot from the Web-UI:

![grafik](https://user-images.githubusercontent.com/19486047/235254690-a7646c79-e15e-433f-b713-b28c05b707e9.png)

**Comment by BkSouX** [comment]
same issue here

**Comment by steve7403** [comment]
same issue here after updating to NC 26.0.1 (from 25.0.6)

I have a 1TB drive for the data directory - about 15 GB are in use. After updating to 26.0.1 i get the warning "Your storage is full, files cannot be updated or synced anymore!". In the web-UI under admin/system/disk, free space shows 24 MB left.

After uploading a file with 500 MB in size, free space switched to a value like -177253 B and the warning message disappeared.

**Comment by joshtrichards** [comment]
Any chance these Synology NAS units are 32-bit?

@steve7403 Are you running on a NAS as well?

**Comment by steve7403** [comment]
@joshtrichards I am running NextcloudPi on an Odroid HC1 which is 32 Bit.
Maybe this is an issue with 32 Bit systems?

**Comment by BkSouX** [comment]
32bits here but not Synology.

**Comment by lanzi69** [comment]
After re-scanning all files the Web-UI now shows the correct size of used space (meanwhile 209,1 GB) but the "storage is full ..." message still appears.

This is the result of the cli command `php occ files:scan lanzi`:

 `http@nextcloud(pts/1):/volume1/web$ nc-php occ files:scan lanzi   `
`Starting scan for user 1 out of 1 (lanzi)                         `
`+---------+-------+--------------+                                `
`| Folders | Files | Elapsed time |                                `
`+---------+-------+--------------+                                `
`| 8075    | 98366 | 01:30:26     |                                `
`+---------+-------+--------------+                                `

**Comment by lanzi69** [comment]
To re-enable syncing I moved about 157 GB to an external folder.

Now my account uses about 52 GB and after re-login the Web-UI shows a message "Your storage is almost full (96%)".

**Comment by lapineige** [comment]
I'm having a similar issue on 32 bit Raspberry Pi 4 with Nextcloud 26.0.0 : https://github.com/nextcloud/server/pull/35734#issuecomment-1541906964

Did you have a patched version including these ? https://github.com/nextcloud/server/pull/35734#issuecomment-1483758053

**Comment by steve7403** [comment]
@lapineige I don't have applied this patch. But after upgrading NC to version 25.0.1 it was not possible to access files anymore. I got the message "folder not found". I changed line 355 in the file (#34905):

/var/www/nextcloud/apps/dav/lib/Connector/Sabre/FilesPlugin.php

from
$propFind->handle(self::SIZE_PROPERTYNAME, function () use ($node): ?int {

to
$propFind->handle(self::SIZE_PROPERTYNAME, function () use ($node) {
            return $node->getSize();

and file access was working again. No Problems then until updating to 26.0.1.

**Comment by lapineige** [comment]
Looks very much like a similar issue to https://github.com/nextcloud/server/pull/35734.
A float becoming an int and everything breaks.

Did you have quota enabled ?

**Comment by MichaIng** [comment]
Strange, I didn't have this issue on my RPi 2 with NC26 when it was in beta, but I still had it on NC25, even after #35734 was merged. Hence I opened #37877 to backport a bunch of related fixes from NC26.

I'll see if I can replicate it now on NC 26.0.1, respectively NC 27 (I'm testing the next beta already), probably another commit broke it again. Devs are quite fast in applying types to mute CI warnings, and do not have 32-bit in mind when doing so 🙈.

**Comment by steve7403** [comment]
@lapineige No, i don't use quota.

The content of the line in FilesPlugin.php is unchanged, so as i modified it.

**Comment by lapineige** [comment]
And if set a quota ?

**Comment by lapineige** [comment]
For my part migrating from 26.0.0 to 26.0.1 doesn't change anything regarding this issue.

**Comment by come-nc** [comment]
Does this fixes it ?
```diff
diff --git a/lib/private/Files/Storage/Local.php b/lib/private/Files/Storage/Local.php
index ee86fe356f7..9af29550662 100644
--- a/lib/private/Files/Storage/Local.php
+++ b/lib/private/Files/Storage/Local.php
@@ -422,7 +422,7 @@ class Local extends \OC\Files\Storage\Common {
                if ($space === false || is_null($space)) {
                        return \OCP\Files\FileInfo::SPACE_UNKNOWN;
                }
-               return (int)$space;
+               return $space;
        }
 
        public function search($query) {
```

**Comment by steve7403** [comment]
@come-nc Yes, free disk space shows the right size with this change.

Many thanks!

**Comment by lapineige** [comment]
It doesn't change anything visually.
I realize now that I no longer have this error message, I don't know why.

I have this error (I don't use a quota):
> [PHP] Error: disk_free_space(): No such file or directory at /var/www/nextcloud/lib/private/Files/Storage/Local.php#421
>
>GET /ocs/v2.php/cloud/users/details?offset=0&limit=25&search=
from IP by USER at TIME

**Comment by ricardomecca** [comment]
Hi,

I have same problem in my nextcloud.
How I can fix this trouble?

**Comment by mmcg180** [comment]
I'm getting this as well after upgrading to 28.0.2

**Comment by ricardomecca** [comment]
> I'm getting this as well after upgrading to 28.0.2

Well, the devs of nextcloud dont reply this! LMAO

**Comment by MichaIng** [comment]
The issue should have been fixed with Nextcloud 27. Are you sure that you really still have space on the storage, and are you running your Nextcloud server on a 32-bit system?

**Comment by mmcg180** [comment]
We use AWS S3 as file storage, local disk storage is at 48%. Running no ubuntu 64-bit

**Comment by MichaIng** [comment]
Then please open a new issue, since your's is unrelated to this one, which affected 32-bit servers only and was fixed, as mentioned.

> Well, the devs of nextcloud dont reply this! LMAO

This is perfectly expected when replying on an old, solved, closed issue. In general, as long as you are not 100% sure that the same bug (with same underlying reason) reappeared, or even when, open a new issue, filling the bug report template. Then you have much better chance to get attention, and we directly have the required information rather than needing to ask, to verify it really is unrelated.

**Comment by ricardomecca** [comment]
> Então abra um novo problema, já que o seu não tem relação com este, que afetou apenas servidores de 32 bits e foi corrigido, conforme referência.
> 
> > Bem, os desenvolvedores do nextcloud não responderam isso! rindo muito
> 
> Isso é perfeitamente esperado ao responder sobre um problema antigo, resolvido e encerrado. Em geral, desde que você não tenha 100% de certeza de que o mesmo bug (com o mesmo motivo subjacente) reapareceu, ou mesmo quando, abra um novo problema, preenchendo o modelo de relatório de bug. Assim, você terá muito mais chances de chamar a atenção e receberemos diretamente as informações possíveis, em vez de precisar perguntar, para verificar se realmente não há relação.

I'm running Nextcloud on Ubuntu SNAP is 64bits, I have 50% space used and the user appears with "unlimited quota" as insufficient space

**Comment by MichaIng** [comment]
Issue has been reported here: #43280
The culprit commit has already been identified: #42591
Should be fixed soon.
